runtime/j9vm/javanextvmi.cpp

```diff
@@ -702,8 +702,8 @@ JVM_CreateThreadSnapshot(JNIEnv *env, jobject thread)
 		vmfns->setCurrentExceptionUTF(currentThread, J9VMCONSTANTPOOL_JAVALANGINTERNALERROR, NULL);
 	} else {
 		j9object_t threadObject = NULL;
-		BOOLEAN isVirtual = JNI_FALSE;
-		BOOLEAN isAlive = JNI_FALSE;
+		BOOLEAN isVirtual = FALSE;
+		BOOLEAN isAlive = FALSE;
```

Comment:
While you're here, could you also please fix line 696: the space should be before `*`, not after like other pointer declarations nearby.

Reply:
Fixed